example.js

```diff
@@ -46,6 +47,14 @@ if (useCustomIcons) {
   MyModeIcon = CustomIcons.CustomModeIcon
 }
 
+// define some application-wide components that should be used in
+// various places
+const components = {
+  ItineraryBody: DefaultItinerary,
```

Comment:
I think `ItineraryFooter` should be mentioned here, at least as a commented-out attribute, so implementers know that it is possible to provide a footer if they so wish.

Reply:
Improved above comment to note ItineraryFooter in 91bdbb5523830976b84ddf35078c2029899d5567.